CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.4.1 (2019-09-27)
+
+- Fixed the select field component to mark the control as touched when the picker is saved. [#49](https://github.com/blackbaud/skyux-select-field/pull/49)
```

Comment:
Wasn't this issue just limited to Firefox?

Reply:
Nope it wasn't. The original issue said it was but I tested and it also affected other browsers.

Reply:
Awesome. Good we got it fixed then!